raiden/smart_contracts/ChannelManagerContract.sol

```diff
@@ -17,6 +17,11 @@ contract ChannelManagerContract {
         uint settle_timeout
     );
 
+    event ChannelDeleted(
+        address caller_address,
+        address partner
+    );
+
     function ChannelManagerContract(address token_address) {
         data.token = Token(token_address);
```

Comment:
Add a wrapper function for `deleteChannel()` here that will wrap the `deleteChannel()` of the Library but also emit the event.